app/policies/division_policy.rb

```diff
@@ -18,4 +18,30 @@ def update?
   def destroy?
     division_admin(division: @record)
   end
+
+  def select?
+    show?
+  end
+
+  class Scope < Scope
```

Comment:
...I'm not seeing how the `show` method is dependent on this new code you've written, but I could be missing something...